src/main/resources/cars.postman_collection.json

```diff
@@ -238,6 +241,177 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "add new car forcing error",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 400\", function () {\r",
+							"    pm.response.to.have.status(400);\r",
+							"});\r",
+							"pm.test(\"response should throw a brand error\", function (){\r",
+							"     pm.expect(pm.response.text()).to.include(\"Brand can't be null\");\r",
+							"});\r",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n    \"country\": \"COUNTRY11111111\"\r\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://localhost:8080/car-api/api/cars",
+					"protocol": "http",
+					"host": [
+						"localhost"
+					],
+					"port": "8080",
+					"path": [
+						"car-api",
+						"api",
+						"cars"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "get car by wrong id",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"no car found with give id\", function () {\r",
+							"\tpm.response.to.have.status(204);\r",
```

Comment:
Better than empty response, what would be more interesting is a message saying what just happened, otherwise the user will be a bit blind not knowing what is wrong with the request.